<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-G09LILE" ignored="1" name="Test(failed)(failed)" tests="2" failures="2" timestamp="07 Apr 2019 16:45:56 GMT" time="28.364" errors="0">
  <testcase name="fillSignUp" time="0.901" classname="com.paratest.testcases.SignUpTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.paratest.pages.SignUpPage.fillSignUpForm(SignUpPage.java:66)
at com.paratest.testcases.SignUpTest.fillSignUp(SignUpTest.java:43)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- fillSignUp -->
  <testcase name="fillSignUp" time="1.631" classname="com.paratest.testcases.SignUpTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.paratest.pages.SignUpPage.fillSignUpForm(SignUpPage.java:66)
at com.paratest.testcases.SignUpTest.fillSignUp(SignUpTest.java:43)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- fillSignUp -->
  <testcase name="anydata" time="0.0" classname="com.paratest.testcases.SignUpTest">
    <ignored/>
  </testcase> <!-- anydata -->
</testsuite> <!-- Test(failed)(failed) -->
